app/views/homepage/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide :page_title, "Science History Institute Digital Collections" %>
```

Comment:
I _think_ you shouldn't have to do this, the title should default to that if you don't provide one. Did you find different?